frameworks/cassandra/src/main/dist/svc.yml

```diff
@@ -164,6 +165,10 @@ pods:
 
                 rm -r container-path/snapshot
         resource-set: sidecar-resources
+        configs:
+          cassandra:
+            template: {{CONFIG_TEMPLATE_PATH}}/cassandra.yaml
```

Comment:
To be clear, is `{{CONFIG_TEMPLATE_PATH}}` still required in `0.30.X`?

Reply:
(just checked the code, and it's still used there).

For reference, this was removed in #1539.